src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -21,5 +21,8 @@
     <type fullname="System.StartupHookProvider" feature="System.StartupHookProvider.IsSupported" featurevalue="false">
       <method signature="System.Boolean get_IsSupported()" body="stub" value="false" />
     </type>
+    <type fullname="System.Resources.ResourceReader" feature="System.Resources.ResourceManager.CustomResourceTypes.AreSupported" featurevalue="false">
```

Comment:
We should update https://github.com/dotnet/runtime/blob/master/docs/workflow/trimming/feature-switches.md with this new switch